PINCache/PINDiskCache.m

```diff
@@ -744,9 +742,18 @@ - (id)objectForKeyedSubscript:(NSString *)key
                 PINDiskCacheError(error);
             }
             [self lock];
-          if (!self->_ttlCache) {
-            [self _locked_setFileModificationDate:now forURL:fileURL];
```

Comment:
- There's one call to `_locked_setFileModificationDate` in `setObject:forKey:fileURL:` that you didn't make async.  Intentional?
- If there are two copies of the same ~10 lines to set this async, worth breaking into a separate method?